OmsAgent/Fairfax/omsagent_ff.py

```diff
@@ -116,11 +118,29 @@ def enable(hutil):
     workspaceKey = protected_settings.get("workspaceKey")
     proxy = protected_settings.get("proxy")
     vmResourceId = protected_settings.get("vmResourceId")
+    stopOnMultipleConnections = protected_settings.get("stopOnMultipleConnections")
```

Comment:
double check with Haitao - I think this is a public setting - we already have discrepancies between Windows and Linux extension - should be careful not to add more (if not required)